templates/taxi/index.html

```diff
@@ -0,0 +1,15 @@
+{% extends "base.html" %}
+
+{% block title %}
+    Taxi Service Overview
+{% endblock %}
+
+{% block content %}
+    <h1>Taxi Service Overview</h1>
+    <ul>
+        <li><strong>Number of cars:</strong> {{ num_cars }}</li>
+        <li><strong>Number of drivers:</strong> {{ num_drivers }}</li>
+        <li><strong>Number of manufacturers:</strong> {{ num_manufacturers }}</li>
+    </ul>
+{% endblock %}
+
```

Comment:
It's a good practice to end HTML files with a blank line for better formatting and to avoid potential issues with some version control systems.